test/e2e/storage/testsuites/subpath.go

```diff
@@ -436,7 +436,7 @@ func (s *subPathTestSuite) DefineTests(driver TestDriver, pattern testpatterns.T
 		testPodFailSubpath(f, l.pod, true)
 	})
 
-	ginkgo.It("should be able to unmount after the subpath directory is deleted", func() {
+	ginkgo.It("should be able to unmount after the subpath directory is deleted [LinuxOnly]", func() {
```

Comment:
Can you add a comment explaining why this doesn't work on Windows

Reply:
added. PTAL